modules/op-about-pipelinerun.adoc

```diff
@@ -3,13 +3,13 @@
 // *openshift_pipelines/op-creating-applications-with-cicd-pipelines.adoc
 
 [id="about-pipelinerun_{context}"]
-= PipelineRun
+= Pipeline run
 
-A _PipelineRun_ is the running instance of a Pipeline. It instantiates a Pipeline for execution with specific inputs, outputs, and execution parameters on a cluster. A corresponding TaskRun is created for each Task automatically in the PipelineRun.
+A _pipeline run_ is the running instance of a pipeline. It instantiates a pipeline for execution with specific inputs, outputs, and execution parameters on a cluster. A corresponding task run is created for each task automatically in the pipeline run.
 
-All the Tasks in the Pipeline are executed in the defined sequence until all Tasks are successful or a Task fails. The `status` field tracks and stores the progress of each TaskRun in the PipelineRun for monitoring and auditing purpose.
+All the tasks in the pipeline are executed in the defined sequence until all tasks are successful or a task fails. The `status` field tracks and stores the progress of each task run in the pipeline run for monitoring and auditing purpose.
 
-The following example shows a PipelineRun to run the `build-and-deploy` Pipeline  with relevant resources and parameters:
+The following example shows a pipeline run to execute the `build-and-deploy` pipeline  with relevant resources and parameters:
```

Comment:
```suggestion
The following example runs the `build-and-deploy` pipeline with relevant resources and parameters:
```
- Per the IBM Style Guide, we should avoid using `execute`. 'Use only if a simpler term, such as "run", "issue", "start", or "enter" is inappropriate.' My suggested edit above shortens the sentence by removing implied meaning and avoids using `execute`. 
- Also, extra space deleted